packages/language-server/test/script.ts

```diff
@@ -0,0 +1,95 @@
+import fs from 'node:fs/promises';
```

Comment:
Is it intended at this time that there's two of this, they're slightly different and are both fork of the script we have in core. When I merge the repo inside the monorepo, I'll update both packages to use the "official" version